### PR TITLE
task 9 code review

### PR DESCRIPTION
**Pull request by saurav-kalsoor-default**

### CODE REVIEW

File: app/src/main/java/com/getdefault/task9/BindingAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package com.getdefault.task9
+
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import androidx.recyclerview.widget.RecyclerView
+import org.w3c.dom.Text
+
+@BindingAdapter("personData")
+fun bindRecyclerview(rv: RecyclerView, list: ArrayList<Person>) {
+    val adapter = rv.adapter as PersonAdapter
```
Comment (rishabh-jain-1990) [inline app/src/main/java/com/getdefault/task9/BindingAdapter.kt:10]: @saurav-kalsoor-default will throw NPE if no adapter is set

File: app/src/main/java/com/getdefault/task9/DisplayViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.getdefault.task9
+
+import androidx.lifecycle.ViewModel
+
+class DisplayViewModel : ViewModel() {
+
+    private val _personList = ArrayList<Person>()
+    val personList: ArrayList<Person>
```
Comment (rishabh-jain-1990) [inline app/src/main/java/com/getdefault/task9/DisplayViewModel.kt:8]: @saurav-kalsoor-default why two variables? You could have achieved the same result by doing a private set()

File: app/src/main/res/layout/person_item.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="10dp"
+        android:background="@color/cardview_dark_background"
+        app:cardCornerRadius="10dp">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="#EFE8A8"
```
Comment (rishabh-jain-1990) [inline app/src/main/res/layout/person_item.xml:23]: @saurav-kalsoor remove all hard coded values

**Review by rishabh-jain-1990 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/person_item.xml (hunk 3 above)
Comment (saurav-kalsoor-default) [reply app/src/main/res/layout/person_item.xml:23]: Ok Sir.

File: app/src/main/java/com/getdefault/task9/BindingAdapter.kt (hunk 1 above)
Comment (saurav-kalsoor-default) [reply app/src/main/java/com/getdefault/task9/BindingAdapter.kt:10]: Ok sir, I'll add checks.

**Pull request merged**